### PR TITLE
Tray unread badge & notification onclick

### PR DESCRIPTION
**Pull request by Zzombiee2361**
I've made a working tray badge, the implementation is a bit hacky but it works. The icon is actually made using canvas in the renderer process, then sent to main process to apply the tray icon. I have yet made message unread counter. Also I overridden the Notification API so all notification will activate the window on click.

### CODE REVIEW

**Comment by Zzombiee2361** [comment]
After some fiddling I'm able to implement the unread counter and event listener using WhatsApp's window.Store. Attaching the event listener is a bit hacky since I don't know when window.Store is loaded, window.onload fires too early.

File: preload.js [hunk 1]
```diff
@@ -23,8 +25,8 @@ window.renderTray = (unread) => {
 		canvas.height = logo.naturalHeight;
 
 		ctx.drawImage(logo, 0, 0);
-		if(unread !== undefined) {
-			unread = (Number(unread) > 99 ? 99 : unread);
+		if(unread > 0) {
+			unread = (unread > 99 ? 99 : unread);
```
Comment (cstayyab) [inline preload.js:29]: Can you display "99+" instead of "99" when Notification count crosses 99?

Comment (Zzombiee2361) [reply preload.js:29]: I thought about that, but it would display 3 characters and won't fit in the badge.

Comment (cstayyab) [reply preload.js:29]: Oh okay, I will leave to 99 for now then an d merge this.

**Pull request merged**